README.md

```diff
@@ -1222,7 +1222,7 @@ Table of Contents
   * [resizeappicon.com](https://resizeappicon.com/) — A simple service to resize and manage your app icons.
   * [Rive](https://rive.app) — Create and ship beautiful animations to any platform. Free forever for Individuals. The service is a editor which hosts all the graphics on their servers as well. They also provide runtimes for many platforms to run graphics made using Rive.
   * [smartmockups.com](https://smartmockups.com/) — Create product mockups, 200 free mockups.
-  * [tablericons.com](https://tablericons.com/) — Free copy and paste SVG editable icons.
+  * [tabler-icons.io](https://tabler-icons.io/) — Over 1500 free copy and paste SVG editable icons.
```

Comment:
these appear to be 2 entirely different services? Are they the same group of people or something?

Reply:
Do you have a copy right on that name? I mean what makes one more official than another? Just asking questions because as an outsider, I have no way of knowing.  Without information to substatiate either I will just remove this entry entirely

Reply:
@angelosds since you added it, any input here?

Reply:
@ripienaar Tabler Icons are MIT-licensed - someone made a site to make money from icons they didn't author - they display ads and promote their own products. 

You can verify it here:
https://www.npmjs.com/package/@tabler/icons
https://github.com/tabler/tabler-icons
https://twitter.com/codecalm